atom/browser/lib/init.coffee

```diff
@@ -22,7 +22,7 @@ if process.platform is 'win32'
   # Redirect node's console to use our own implementations, since node can not
   # handle console output when running as GUI program.
   print = (args...) ->
-    process.log util.format(args...)
+    process.log util.format(args...) + "\n"
   console.log = console.error = console.warn = print
   process.stdout.write = process.stderr.write = print
```

Comment:
The `process.stdout.write` should behave the same with `std::wcout`, e.g. not formatting the arguments and append no new line. And we can also make it accept `Buffer` as input as mentioned in #2390.